docker/Dockerfile-pgguardian

```diff
@@ -0,0 +1,2 @@
+FROM pulseguardian_pulseguardian-base
```

Comment:
Another way of implementing this file and `docker/Dockerfile-pgweb` would be to move the contents to `docker-compose.yml` as additional entries with differing `environment` and `command` keys. Though subjective style choice so doesn't matter either way :-)